ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,227 @@
+Type: Process
+
+Created: 2022-14-03
+
+## What is ZEP?
+
+ZEP stands for Zarr Enhancement Proposal. A ZEP is a design document providing information to the Zarr community, describing a modification or enhancement of the Zarr storage specification, a new feature for Zarr or its processes or environment. The ZEP should provide specific proposed changes to the Zarr specification and a narrative rationale for the specification changes. For feature changes, the ZEP should provide a concise technical specification and a rationale for the feature.
+
+We intend ZEPs to be the primary mechanisms for evolving the spec, proposing major new features, collecting community input on major issues and documenting the design decision that has gone into Zarr. The ZEP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Because the ZEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+## ZEP audience
+
+The typical primary audience for ZEPs is the developers working on Zarr and its various implementations, the Zarr steering council as well as the Zarr community.
+
+The broader Zarr community may also choose to use the process to document expected feature additions and to manage complex design coordination problems that
+require collaboration across multiple projects.
+
+## ZEP types
+
+> Specification ZEP
+
+Specification ZEPs deal with the changes related to Zarr spec. The SPEC ZEP could introduce changes in core specification or can add changes as an extension to the core protocol. The changes related to core specification follow a strict and thorough review process and should be adopted by everyone in the Zarr community, whereas changes made as an extension to the core protocol follow a comparatively simple review process and is not strictly necessary to be adopted by everyone in the Zarr community. The specification ZEPs are of two types:
+
+- Core protocol ZEP
+
+Describes a ZEP which involves changes in the core specification protocol for Zarr. Core protocol ZEPs(commonly known as Core ZEPs) are a part of Spec ZEPs and apply to [Zarr storage specification](https://github.com/zarr-developers/zarr-specs) and its various implementations under the [Zarr Implementations](https://github.com/zarr-developers/zarr_implementations) GitHub repo. The core protocol ZEPs should be adopted by every implementation of Zarr and, in general, the overall Zarr community. Core ZEPs must go through a thorough review process, including involvement, discussion, and voting from the author of the proposed ZEP, Zarr Steering Council, author of various Zarr implementations, and open-source projects/research groups using Zarr and the general Zarr community. The general advice is that everyone should be made aware of the change by the introduction of core ZEP. Core protocol ZEP require community consensus, and users are typically not free to ignore them.
```

Comment:
What do we mean by "involvement" on this line?

Reply:
This mostly means getting them involved in the discussion and voting process by means of creating issues or opening discussion threads or using gitter.